backend/test/mocks/MafiaGameMock.js

```diff
@@ -45,51 +49,67 @@ module.exports.createMafiaGameWithOnePlayerMock = function (port) {
     });
 
     return { io, mafiaGame, socketIOServer: server, roomID, hostPlayer };
-};
+}
 
-module.exports.addPlayer = function (player, roomID) {
+function addPlayer(player, roomID) {
     let room = mafiaGame.gameRoomsDict[roomID];
 
     room.addPlayer(player);
-};
+}
+
+function addPlayers(players, roomID) {
+    const room = mafiaGame.gameRoomsDict[roomID];
+    for (let player of players) {
+        room.addPlayer(player);
+    }
+}
 
-module.exports.addMafiaVote = function (voter, votedFor, roomID) {
+function addMafiaVote(voter, votedFor, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     const { mafiaVoteMap } = room.voteHandler;
 
     mafiaVoteMap[voter.nickname] = votedFor;
-};
+}
 
-module.exports.addDayVote = function (voter, votedFor, roomID) {
+function addDayVote(voter, votedFor, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     const { daytimeVoteMap } = room.voteHandler;
 
     daytimeVoteMap[voter.nickname] = votedFor;
-};
+}
 
-module.exports.addTrialVote = function (voter, votedFor, roomID) {
+function addTrialVote(voter, votedFor, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     const { trialVoteMap } = room.voteHandler;
 
     trialVoteMap[voter.nickname] = votedFor;
-};
-
-module.exports.addPlayers = function (players, roomID) {
-    const room = mafiaGame.gameRoomsDict[roomID];
-    for (let player of players) {
-        room.addPlayer(player);
-    }
-};
-
-module.exports.switchPlayer = function (nickname, roomID) {
+}
+
+/**
+ * Switch the current player (socket.player)
+ * @param {*} nickname
+ * @param {*} roomID
+ */
+function switchPlayer(nickname, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     testPlayerSocket.player = room.getPlayerByNickname(nickname);
-};
+}
 
-module.exports.resetRoom = function (roomID) {
+function resetRoom(roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     room.resetGame();
     room.players = [];
     room.host = null;
     room.voteHandler.resetVotes();
+}
+
+module.exports = {
```

Comment:
Refactored this a bit, to be consistent with how we've done exports in other test files.